app/assets/javascripts/front/realization.js

```diff
@@ -0,0 +1,283 @@
+      this._scrollTo(this.target)
+    })
+
+    this._scrollTo(this.target)
+  }
+
+  loadMobilePlayer() {
+    let jwConfig = {
+      "aspectratio": "9:16",
+      "autostart": false,
+      "cast": {
+        "appid": "00000000"
+      },
+      "controls": true,
+      "displaydescription": false,
+      "displaytitle": false,
+      "flashplayer": "//ssl.p.jwpcdn.com/player/v/8.17.7/jwplayer.flash.swf",
+      "height": 360,
+      "key": "G4TQ74H6dL+J2s9zjsY53y72/VJdEKk5UO4K8F3CL4mUDDFs7Wl6siOex5c=",
+      "mute": false,
+      "ph": 3,
+      "pid": "NaHiexhY",
+      "playbackRateControls": false,
+      "playlist": "//cdn.jwplayer.com/v2/media/8FivThod?recommendations_playlist_id=xFiJh22s",
+      "preload": "metadata",
+      "repeat": false,
+      "stagevideo": false,
+      "stretching": "uniform",
+      "width": "100%"
+    }
+    jwplayer("mobileVideo").setup(jwConfig)
+
+    let player = jwplayer("mobileVideo")
+    this.target = "video"
+
+    player.on('pause', () => {
+      this.target = "paused"
+      this.container.dataset.screen = this.target
+      this._scrollTo(this.target)
+    })
+
+    player.on('complete', () => {
+      this.target = "survey"
+      this.container.dataset.screen = this.target
+      this._scrollTo(this.target)
+    })
+
+    this._scrollTo(this.target)
+  }
+
+  loadJWPlayer(div) {
+    let player = jwplayer(div)
+    this.target = "video"
+
+    player.on('pause', () => {
+      this.target = "paused"
+      this.container.dataset.screen = this.target
+      this._scrollTo(this.target)
+    })
+
+    player.on('complete', () => {
+      this.target = "survey"
+      this.container.dataset.screen = this.target
+      this._scrollTo(this.target)
+    })
+
+    this._scrollTo(this.target)
+  }
+
+  passFeel() {
+    let feelings = []
+    let feelFlag = false
+    for (let i = 0; i < this.surveyBtns.length; i++) {
+      let surveyBtn = this.surveyBtns[i]
+      if (surveyBtn.classList.contains('button--active')) {
+        feelFlag = true
+        feelings.push(surveyBtn.id)
+      }
+    }
+    if(feelFlag) {
+      let props = {
+        "feel": feelings
+      }
+      this.target = "course"
+      this.container.dataset.screen = this.target
+      this._scrollTo(this.target)
+
+      this.courseFeelInput.value = JSON.stringify(props)
+    } else {
+      $(this.surveyMsg).show()
+    }
+  }
+
+  formSending(btn) {
+    btn.innerText = "Loading..."
+  }
+
+  formSuccess() {
+    this.target = "thanks"
+    this.container.dataset.screen = this.target
+  }
+
+  formError() {
+    console.log("Error in form submission...")
+  }
+
+  share() {
+    const shareData = {
+      title: document.title,
+      text: 'Experience Self Realization',
+      url: window.location.href
+    }
+    if (navigator.share) {
+      navigator.share(shareData)
+        .then(() => console.log('Successful share'))
+        .catch((error) => console.log('Error sharing...', error))
```

Comment:
@Mrinalsinghal You can add this comment `// eslint-disable-line no-console` to the end of this line to suppress the error. Since error reporting is needed. You should probably also change it to `console.error` instead of `console.log`